mteb/tasks/BitextMining/multilingual/NTREXBitextMining.py

```diff
@@ -75,8 +267,8 @@ class NTREXBitextMining(AbsTaskBitextMining, CrosslingualTask):
         annotations_creators="expert-annotated",
         dialect=[],
         text_creation="created",
-        n_samples={"train": _N},
-        avg_character_length={"train": 120},
+        n_samples={"test": _N},
```

Comment:
I believe this is _N x number of language pairs? (for the whole dataset)

Reply:
You're right, will change it to `_N*len(_EVAL_LANGS)` (`EVAL_LANGS` has 1916 language directions)